src/Files.App/Actions/Show/ToggleSidebarAction.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) 2024 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.App.Actions
+{
+	internal sealed class ToggleSidebarAction : IToggleAction
```

Comment:
```suggestion
	internal sealed class ToggleSidebarAction : ObservableObject, IToggleAction
```

Reply:
I forgot toggle actions use on property changed.

Reply:
I reverted it.